src/platforms/common/data-management/sensitive-data/attachment-scrubbing.mdx

```diff
@@ -0,0 +1,92 @@
+  Generally they will contain binary data so be careful with the [data types](../advanced-datascrubbing/#data-types) used to scrub these regions.
+
+  In particular for stack memory it is **not recommended** to be scrubbed, since this is used by the event processing pipeline to unwind the stack and build a readable stack trace for the event.
+  Inadvertedly modifying the stack in an undesired way will make this process impossible and the event much less useful.
+
+  In general all [data types](../advanced-datascrubbing/#data-types) result in a textual regular expression match on some data, for the stack this regular expression is matched as a binary UTF-8 regular expression so that any UTF-8 embedded in the binary data will be matched.
+  Additionally the regular expression is applied to any valid UTF-16LE strings found within the binary data.
+
+- The **heap memory**: For the current implementation this is considered to be any memory region which is not used as stack memory.
+  This means it could include other non-stack regions like `mmap`ed files.  All these memory regions can be selected using the `$heap_memory` [value type](../advanced-datascrubbing/#value-types) selector.
+
+  Since the heap memory is not used in the event processing pipeline it is much safer to modify these memory regions.
+
+  Like for stack memory all [data types](../advanced-datascrubbing/#data-types) are matched as a binary UTF-8 regular expression as well as in any valid UTF-16LE strings found within the binary memory region.
+
+- A minidump also contains a list of **code modules** which were loaded at the time the minidump was created.
+  This module list contains information about each module, including two fields which could contain file paths: the path of the code file and the path of the associated debug file.
+
+  As these fields are pathnames they could contain a user's home directory, so it makes sense to apply the *Usernames in filepaths* [data type](../advanced-datascrubbing/#data-types) to these fields.  The fields can be selected individually by their respective field names:
+
+  - `code_file`
+  - `debug_file`
+
+  Both fields have a [value type](../advanced-datascrubbing/#value-types) of `$string`.
+  Any modification will exclude the basename of the pathnames to ensure the event processing pipeline can still operate correctly on the minidump.
+
+  An example of a rule to scrub these fields would be displayed by the UI like this:
+
+  ```
+  [Remove] [Usernames in filepaths] from [$minidump.code_file || $minidump.debug_file]
+  ```
+
+- Linux minidumps can contain the process's **command line** and **environment variables**, these are included as copies of the files in `/proc/$pid/cmdline` and `/proc/$pid/environ`.
+  As they are direct copies of these files, they have this file format: NULL-separated records of `argv` arguments or `VAR=val` pairs respectively.
```

Comment:
```suggestion
- Linux minidumps can contain both the **command line** and **environment variables**, which are included as copies of the files in `/proc/$pid/cmdline` and `/proc/$pid/environ`. Since these are direct copies of these files, they have this file format: NULL-separated records of `argv` arguments or `VAR=val` pairs respectively.
```